dubbo-common/src/main/java/org/apache/dubbo/common/io/Bytes.java

```diff
@@ -730,8 +730,12 @@ public static String bytes2base64(final byte[] bs, final int off, final int len,
             char pc = code[64];
             if (str.charAt(off + len - 2) == pc) {
                 size -= 2;
+                --num;
```

Comment:
it looks like we can unify the logic for method *public static byte[] base642bytes(final String str, final int off, final int len, final String code)* and method *public static byte[] base642bytes(final String str, final int off, final int len, final char[] code) *

would you mind to give it a try?